### ISSUE TITLE
Fusion Rate

### ISSUE THREAD

**Issue by luizamanfrini** [kriswiner/MPU-9250#420]
Hi Kris! I am new to Madgwick and I am a bit confused with que fusion rate. How can I define it and change it?

Thank you for your attention. Your code is great!

**Comment by kriswiner** [comment]
The Madgwick filter is iterative, so generally 10 - 20 iterations pre new
data set is enough to get good stable yaw estimation. So the rate will
depend on how fast your MCu is. It is not something you can adjust except
by changing your processor.

See this example:

https://github.com/kriswiner/MPU9250/tree/master/AK8963_as_slave

**Comment by basil96** [comment]
Hi Kris,

I'm getting ~6170 Hz fusion rate on the Dragonfly, is that even real? Seems unbelievably high!

BTW, I had to make the same change to the `Wire.begin()` call in `MPU9250BasicAHRS_Dragonfly` as discussed in #421.  Can I submit that as a PR?

**Comment by kriswiner** [comment]
Yes, this is correct. The L4 is a Cortex M4F, meaning it has a FPU. You
really only need to iterate 10 or 20 times for best results with the
Madgwick or Mahoney filters, so if you are at 6 kHz fusion rate you could
enforce the iteration rate to 10 or 20 times or increase the data sample
rate to 500 or 1000 Hz.

**Comment by basil96** [comment]
Awesome. I can do that.
I just noticed that you committed the I2C change yesterday, nvm :)
